### DISCUSSION TITLE
"Installing WIndows Server" trying to upgrade from Win10Pro 20H2...

### DISCUSSION THREAD

**Discussion by LenCardinal** [AveYo/MediaCreationTool.bat#73] [Q&A] [answered]
I may not have done enough homework...but I ran MediaCreationTool.bat on a laptop with Win10 Pro 20H2, and chose _Windows 11_, then _Auto Update_. Expecting to upgrade to Win11 Pro.

The machine is now saying "Installing Windows Server". Is this supposed to happen?

**Comment by AveYo** [comment, marked as answer]
Yes, it's in the front readme  
If your system passes windows 11 requirements, then you can rename the script as `def MediaCreationTool.bat` and it will disable the built-in requirements bypass that changes the setup label..

**Reply by LenCardinal** [reply]
Got it, thanks.

Awesome tool.
